docs/features/pootle_fs_statuses.rst

```diff
@@ -21,38 +23,37 @@ to resolve them:
 
 ``conflict``
   Both the Pootle store and the filesystem file have changed. To resolve this
-  conflict use :djadmin:`merge` to merge the two and manage conflict
-  resolution. Or if you wish to discard one or the other use :option:`fetch
-  --force <fetch --force>` or :option:`add --force <add --force>` to keep the
-  filesystem version or Pootle version respectively.
+  conflict use :djadmin:`resolve` to merge the two and manage conflict
+  resolution. Or if you wish to discard one or the other use :option:`resolve
+  --overwrite`.
 
 ``conflict_untracked``
   Conflict can also arise if both the store and file are untracked. In this
   case you can use :djadmin:`merge` to combine the translation and manage
   conflict resolution for each unit. Or to force taking the whole file or store
-  use either :option:`fetch --force <fetch --force>` or :option:`add --force
-  <add --force>` depending on whether you want to keep the filesystem file or
-  the Pootle store.
+  use either :option:`fetch --force` or :option:`add --force` depending on
+  whether you want to keep the filesystem file or the Pootle store.
 
 ``pootle_untracked``
   A new store has been added in Pootle but does not have any matching file on
   the filesystem. You can use either :djadmin:`add` to create the file on the
   filesystem and push the translations on the store to it, or alternatively use
-  :djadmin:`rm` to stage the store for removal.
+  :option:`rm --force` to stage the store for removal.
```

Comment:
This should be `:command:`

Reply:
@unho no it should not, it should be `:option:`